policy/modules/system/systemd.fc

```diff
@@ -44,6 +44,13 @@
 /usr/lib/systemd/systemd-user-sessions	--	gen_context(system_u:object_r:systemd_sessions_exec_t,s0)
 
 # Systemd unit files
+HOME_DIR/\.config/systemd(/.*)?		gen_context(system_u:object_r:systemd_user_home_t,s0)
+HOME_DIR/\.config/systemd/user(/.*)?		gen_context(system_u:object_r:systemd_user_home_unit_t,s0)
+HOME_DIR/\.local/share/systemd(/.*)?		gen_context(system_u:object_r:systemd_user_home_t,s0)
+HOME_DIR/\.local/share/systemd/user(/.*)?		gen_context(system_u:object_r:systemd_user_home_unit_t,s0)
+
```

Comment:
I think this might be overdoing it. just a systemd_conf_home_t and systemd_data_home_t should probably be reasonable enough. The units are only systemd_conf_home_t i believe and the systemd_data_home_t is used to store timer stamp files if i am not mistaken.

Reply:
Okay that makes sense. Should we also just make system-wide user units `systemd_unit_t` instead of a separate `systemd_user_unit_t` type?